Add backslash-escaping to bare - used as minus in man page.
--- a/cwmrc.5
+++ b/cwmrc.5
@@ -186,7 +186,7 @@
 .It Ic htile Ar percent
 Set the percentage of screen the master window should occupy
 after calling
-.Ic window-htile .
+.Ic window\-htile .
 If set to 0, the horizontal size of the master window will
 remain unchanged.
 The default is 50.
--- a/cwm.1
+++ b/cwm.1
@@ -82,65 +82,65 @@
 The default key bindings are:
 .Pp
 .Bl -tag -width "CM-EscapeXXXXX" -offset indent -compact
-.It Ic CM-Return
+.It Ic CM\-Return
 Spawn a new terminal.
-.It Ic CM-Delete
+.It Ic CM\-Delete
 Lock the screen.
-.It Ic M-Return
+.It Ic M\-Return
 Hide current window.
-.It Ic M-Down
+.It Ic M\-Down
 Lower current window.
-.It Ic M-Up
+.It Ic M\-Up
 Raise current window.
-.It Ic M-slash
+.It Ic M\-slash
 Search for windows.
-.It Ic C-slash
+.It Ic C\-slash
 Search for applications.
-.It Ic CM-n
+.It Ic CM\-n
 Label current window.
-.It Ic M-Tab
+.It Ic M\-Tab
 Cycle through currently visible windows.
-.It Ic MS-Tab
+.It Ic MS\-Tab
 Reverse cycle through currently visible windows.
-.It Ic CM-x
+.It Ic CM\-x
 Close current window.
-.It Ic CM-[n]
-Toggle visibility of group n, where n is 1-9.
-.It Ic CM-a
+.It Ic CM\-[n]
+Toggle visibility of group n, where n is 1\(en9.
+.It Ic CM\-a
 Toggle visibility of all groups.
-.It Ic CM-g
+.It Ic CM\-g
 Toggle group membership of current window.
-.It Ic M-Right
+.It Ic M\-Right
 Cycle through active groups.
-.It Ic M-Left
+.It Ic M\-Left
 Reverse cycle through active groups.
-.It Ic CMS-f
+.It Ic CMS\-f
 Toggle freezing geometry of current window.
-.It Ic CM-s
+.It Ic CM\-s
 Toggle stickiness of current window.
-.It Ic CM-f
+.It Ic CM\-f
 Toggle full-screen mode of current window.
-.It Ic CM-m
+.It Ic CM\-m
 Toggle maximization of current window.
-.It Ic CM-equal
+.It Ic CM\-equal
 Toggle vertical maximization of current window.
-.It Ic CMS-equal
+.It Ic CMS\-equal
 Toggle horizontal maximization of current window.
-.It Ic M-[hjkl]
+.It Ic M\-[hjkl]
 Move window by a small amount.
-.It Ic MS-[hjkl]
+.It Ic MS\-[hjkl]
 Move window by a large amount; see
 .Xr cwmrc 5 .
-.It Ic CM-[hjkl]
+.It Ic CM\-[hjkl]
 Resize window by a small amount.
-.It Ic CMS-[hjkl]
+.It Ic CMS\-[hjkl]
 Resize window by a large amount; see
 .Xr cwmrc 5 .
-.It Ic M-question
+.It Ic M\-question
 Spawn
 .Dq exec program
 dialog.
-.It Ic M-period
+.It Ic M\-period
 Spawn
 .Dq ssh to
 dialog.
@@ -149,46 +149,46 @@
 to provide host auto-completion.
 .Xr ssh 1
 will be executed via the configured terminal emulator.
-.It Ic CM-w
+.It Ic CM\-w
 Spawn
 .Dq exec WindowManager
 menu, allowing a switch to another window manager.
-.It Ic CMS-r
+.It Ic CMS\-r
 Restart.
-.It Ic CMS-q
+.It Ic CMS\-q
 Quit.
 .El
 .Pp
 The default mouse bindings are:
 .Pp
-.Bl -tag -width "CM-EscapeXXXXX" -offset indent -compact
-.It Ic M-M1
+.Bl -tag -width "CM\-EscapeXXXXX" -offset indent -compact
+.It Ic M\-M1
 Move current window.
-.It Ic CM-M1
+.It Ic CM\-M1
 Toggle group membership of current window.
-.It Ic M-M2
+.It Ic M\-M2
 Resize current window
-.It Ic M-M3
+.It Ic M\-M3
 Lower current window.
-.It Ic CMS-M3
+.It Ic CMS\-M3
 Hide current window.
 .El
 .Pp
 The following key bindings may be used to navigate
 search and exec dialogs:
 .Pp
-.Bl -tag -width "[Down] or C-s or M-j" -offset indent -compact
+.Bl -tag -width "[Down] or C\-s or M\-j" -offset indent -compact
 .It Ic [Return]
 Select item.
-.It Ic [Down], C-s No or Ic M-j
+.It Ic [Down], C\-s No or Ic M\-j
 Next item.
-.It Ic [Up], C-r No or Ic M-k
+.It Ic [Up], C\-r No or Ic M\-k
 Previous item.
-.It Ic [Backspace] No or Ic C-h
+.It Ic [Backspace] No or Ic C\-h
 Backspace.
-.It Ic C-u
+.It Ic C\-u
 Clear input.
-.It Ic C-a
+.It Ic C\-a
 List all available items.
 .It Ic [Esc]
 Cancel.
@@ -271,7 +271,7 @@
 .Nm
 was originally inspired by evilwm, but was rewritten from scratch
 due to limitations in the evilwm codebase.
-The from-scratch rewrite borrowed some code from 9wm, however that code
+The from\-scratch rewrite borrowed some code from 9wm, however that code
 has since been removed or rewritten.
 .Pp
 .Nm
